src/main/scala/nl.knaw.dans.easy.bagstore/BagFacadeComponent.scala

```diff
@@ -146,9 +146,9 @@ trait Bagit4FacadeComponent extends BagFacadeComponent {
 
     override def getPayloadFilePaths(bagDir: Path): Try[Set[Path]] =  {
       getBag(bagDir)
-        .map(_.getPayloadManifests.asScala.headOption.map(_.asScala.map {
+        .map(_.getPayloadManifests.asScala.map(_.asScala.map {
           case (path, _) => Paths.get(path)
-        }.toSet).getOrElse(Set.empty))
+        }.toSet)).map(_.reduce(_ ++ _))
```

Comment:
Well, the assumption is that the bag is virtually-valid, otherwise the result of is unreliable anyway, as the funcion advertises to return the payload file paths, and doesn't mention that it implements this using the payload manifests. I'll document this requirement just to be sure.